ui/components/component-library/text/README.mdx

```diff
@@ -0,0 +1,328 @@
+
+<Text color={COLORS.TEXT_DEFAULT}>
+  text-default
+</Text>
+<Text color={COLORS.TEXT_ALTERNATIVE}>
+  text-alternative
+</Text>
+<Text color={COLORS.TEXT_MUTED}>
+  text-muted
+</Text>
+<Text color={COLORS.OVERLAY_INVERSE} backgroundColor:{COLORS.OVERLAY_DEFAULT}>
+  overlay-inverse
+</Text>
+<Text color={COLORS.PRIMARY_DEFAULT}>
+  primary-default
+</Text>
+<Text color={COLORS.PRIMARY_INVERSE} backgroundColor:{COLORS.PRIMARY_DEFAULT}>
+  primary-inverse
+</Text>
+<Text color={COLORS.SECONDARY_DEFAULT}>
+  secondary-default
+</Text>
+<Text color={COLORS.SECONDARY_INVERSE} backgroundColor:{COLORS.SECONDARY_DEFAULT}>
+  secondary-inverse
+</Text>
+<Text color={COLORS.ERROR_DEFAULT}>
+  error-default
+</Text>
+<Text color={COLORS.ERROR_INVERSE} backgroundColor:{COLORS.ERROR_DEFAULT}>
+  error-inverse
+</Text>
+<Text color={COLORS.SUCCESS_DEFAULT}>
+  success-default
+</Text>
+<Text color={COLORS.SUCCESS_INVERSE} backgroundColor:{COLORS.SUCCESS_DEFAULT}>
+  success-inverse
+</Text>
+<Text color={COLORS.WARNING_INVERSE} backgroundColor:{COLORS.WARNING_DEFAULT}>
+  warning-inverse
+</Text>
+<Text color={COLORS.INFO_DEFAULT}>
+  info-default
+</Text>
+<Text color={COLORS.INFO_INVERSE} backgroundColor:{COLORS.INFO_DEFAULT}>
+  info-inverse
+</Text>
+```
+
+### Font Weight
+
+Use the `fontWeight` prop and the `FONT_WEIGHT` object from `./ui/helpers/constants/design-system.js` to change the font weight of the Typography component. There are 2 font weights:
+
+- `FONT_WEIGHT.NORMAL` = `normal` || `400`
+- `FONT_WEIGHT.MEDIUM` = `medium` || `500`
+- `FONT_WEIGHT.BOLD` = `bold` || `700`
+
+<Canvas>
+  <Story id="ui-components-component-library-text-text-stories-js--font-weight" />
+</Canvas>
+
+```jsx
+// If importing from ui/components/app/[YOUR_COMPONENT]/ directory
+import Text from '../../ui/component-library/text';
+import { FONT_WEIGHT } from '../../../helpers/constants/design-system';
+
+<Text fontWeight={FONT_WEIGHT.NORMAL}>
+  normal
+</Text>
+<Text fontWeight={FONT_WEIGHT.MEDIUM}>
+  medium
+</Text>
+<Text fontWeight={FONT_WEIGHT.BOLD}>
+  bold
+</Text>
+```
+
+### Font Style
+
+Use the `fontStyle` prop and the `FONT_STYLE` object from `./ui/helpers/constants/design-system.js` to change the font style of the Typography component. There are 2 font styles:
+
+- `FONT_STYLE.NORMAL`
+- `FONT_STYLE.ITALIC`
+
+<Canvas>
+  <Story id="ui-components-component-library-text-text-stories-js--font-style" />
+</Canvas>
+
+```jsx
+// If importing from ui/components/app/[YOUR_COMPONENT]/ directory
+import Text from '../../ui/component-library/text';
+import { FONT_STYLE } from '../../../helpers/constants/design-system';
+
+<Text fontStyle={FONT_STYLE.NORMAL}>
+  normal
+</Text>
+<Text fontStyle={FONT_STYLE.ITALIC}>
+  bold
+</Text>
+```
+
+### Align
+
+Use the `align` prop and the `TEXT_ALIGN` object from `./ui/helpers/constants/design-system.js` to change the text alignment of the Typography component.
```

Comment:
Updating component name and also removing full stop from end of sentence I think we should adopt the content guidelines of the app - Sentence case everything(with some exceptions) only punctuation on sentences that have proceeding sentences and no full stop at the end of the last sentence

```suggestion
Use the `align` prop and the `TEXT_ALIGN` object from `./ui/helpers/constants/design-system.js` to change the text alignment of the `Text` component
```